sys/shell/commands/shell_commands.c

```diff
@@ -214,5 +218,8 @@ const shell_command_t _shell_command_list[] = {
     { "ccnl_cont", "create content and populated it", _ccnl_content},
     { "ccnl_fib", "shows or modifies the CCN-Lite FIB", _ccnl_fib},
 #endif
+#ifdef MODULE_SNTP
+    { "ntpdate", "synchronizes with a remote time server", _ntpdate},
```

Comment:
I guess this should have been a reply to https://github.com/RIOT-OS/RIOT/pull/6020#discussion_r85656944, so yes (though I would check for `argc > 2`, instead of `argc == 3`). Also style ;-). And the 5000 should be a constant instead of a literal.